composer/workflows/pythonvirtualenvoperator_python2_test.py

```diff
@@ -0,0 +1,31 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import sys
+
+import pytest
+
+from . import unit_testing
+
+
+@pytest.mark.skipif(sys.version_info >= (3, 0), reason="requires Python 2")
```

Comment:
It seems like the DAG should be importable from Python3 like the other one in this commit, unless I'm missing something

Reply:
You're right. https://github.com/GoogleCloudPlatform/python-docs-samples/pull/1714